### PR TITLE
add types to thumbnail driver

### PR DESCRIPTION
**Pull request by lbelinsk**

### CODE REVIEW

File: packages/wix-ui-core/src/components/StylableText/Text.driver.ts [hunk 1]
```diff
@@ -12,7 +21,7 @@ export const textDriverFactory = ({element}) => {
     /** check if element has title attribute */
     hasTitleAttribute: () => element.getAttribute('title') !== null,
     /** check if element has title attribute */
-    getTitle: () => element.title,
+    getTitle: () => (element as HTMLElement).title,
```
Comment (hepiyellow) [inline packages/wix-ui-core/src/components/StylableText/Text.driver.ts]: Can you use the `<HTMLElement>element.title` variation, like you did in Thumbnail?

File: packages/wix-ui-core/src/testkit/index.ts [hunk 2]
```diff
@@ -25,8 +25,9 @@ export const checkboxTestkitFactory = testkitFactoryCreator(checkboxDriverFactor
 import {toggleSwitchDriverFactory} from '../components/ToggleSwitch/ToggleSwitch.driver';
 export const toggleSwitchTestkitFactory = testkitFactoryCreator(toggleSwitchDriverFactory);
 
-import {textDriverFactory as stylableTextDriverFactory} from '../components/StylableText/Text.driver';
+import {textDriverFactory as stylableTextDriverFactory, TextDriver} from '../components/StylableText/Text.driver';
 export const stylableTextTestkitFactory = testkitFactoryCreator(stylableTextDriverFactory);
```
Comment (hepiyellow) [inline packages/wix-ui-core/src/testkit/index.ts]: You can add `testkitFactoryCreator<TextDriver>`

Comment (lbelinsk) [reply packages/wix-ui-core/src/testkit/index.ts]: yes, missed that, thanks!

File: packages/wix-ui-core/src/components/StylableText/Text.driver.ts (hunk 1 above)
Comment (lbelinsk) [reply packages/wix-ui-core/src/components/StylableText/Text.driver.ts]: No. it didn't work

**Pull request merged**